README.md

```diff
@@ -1,67 +1,76 @@
-# Kubernetes-KeyVault-FlexVolume #
+# Key Vault FlexVolume
 
-Key Vault FlexVolume for Kubernetes - Integrates Key Management Systems with Kubernetes via a FlexVolume.
+Seamlessly integrate your key management systems with Kubernetes.
 
-With the Key Vault FlexVolume, developers can mount multiple secrets, keys, and certs stored in Key Management Systems into their pods as a volume. Once the Volume is attached, the data in it is mounted into the container's file system.
+Secrets, keys, and certificates in a key management system become a volume accessible to pods. Once the volume is mounted, its data is available directly in the container filesystem for your application.
```

Comment:
`Secrets, keys, and certificates in a key management system can be mounted as a volume accessible to pods`

Reply:
nvm the diffs got to me :)